SimpleConsoleApplication/Program.cs

```diff
@@ -4,6 +4,11 @@
 double doubleExample = 3.3333333D;
 char charExample = 'C';
 
+bool IsEven(int integer)
+{
+    return integer % 2 == 0;
+}
+
```

Comment:
Would it have been better to have declared this function in a class, or even a separate class library to be imported? Not quite sure if that's what the moduralisation AC on the task was hinting at.

Reply:
@msmi1433 That will come later on in the tasks to do, at this point is not required to be in a separate file or class.